networking-calico/networking_calico/plugins/ml2/drivers/calico/test/lib.py

```diff
@@ -586,6 +621,12 @@ def port_query(self, **kw):
                 if fip['fixed_port_id'] == kw['fixed_port_id']:
                     fips.append(fip)
             return fips
+        elif kw.get('id', None):
+            for network in self.osdb_networks:
+                if network['id'] == kw['id']:
+                    network_mock = mock.MagicMock()
+                    network_mock.first.return_value = network
+                    return network_mock
```

Comment:
I think we should now rename `port_query` to `db_query` (or similar) and add comments here to say that:
- the 'port_id' case occurs when querying for IPAllocations
- the 'fixed_port_id' case occurs when querying for FloatingIPs
- the 'id' case occurs when querying for Networks.

Reply:
Changed name to more accurately reflect that method now.